Errant

Nowordsandnotune opened the pull request:
I did the thing!

Nowordsandnotune committed:
Create moo.md

AAAAAAAAAAAAH

Nowordsandnotune committed:
Update and rename moo.md to Mulch.md

Nowordsandnotune committed:
Update Mulch.md

Nowordsandnotune committed:
Update Mulch.md

Nowordsandnotune committed:
Update Mulch.md

malpinder commented on pals/Mulch.md:
```diff
@@ -0,0 +1,69 @@
+* Sleep (Spell, Daily)
+* Make Whole (Ritual)
+* Silence (Ritual)
+* Animal Messenger (Ritual)
```
FYI: Riteby also has Silence, and Jo's cleric has Make Whole. There's no harm in doubling up (there's not many rituals at level 1 😞 ) but you might be interested in Brew Potion, which was changed to level 1 in errata, or Wizard's Curtain or Alarm from the old dragon magazine.

Nowordsandnotune committed:
Update Mulch.md

malpinder committed:
Update Mulch.md

malpinder committed:
Update Mulch.md

malpinder committed:
Remove some redunant information

Mulch can detect magic and does have the orb of imposition feature, but they're already recorded elsewhere on the character sheet.
